src/stuf/stuf_zds/templates/stuf_zds/soap/genereerDocumentIdentificatie.xml

```diff
@@ -1,11 +1,12 @@
-<ZKN:genereerDocumentIdentificatie_Di02
+{% extends "stuf/soap_envelope.xml" %}{% load stuf %}
+{% block body %}<ZKN:genereerDocumentIdentificatie_Di02
         xmlns:StUF="http://www.egem.nl/StUF/StUF0301"
```

Comment:
namespace mess - StUF-BG defines them on the root element, while StUF-ZDS defines them on the children? Why this inconsistency?

Reply:
Resolved via 0126688d73541bbe5ff157b1114d35cc71426dec